eq-author-api/schema/typeDefs.js

```diff
@@ -43,6 +45,7 @@ type Questionnaire {
   shortTitle: String
   displayName: String!
   introduction: QuestionnaireIntroduction
+  isNew: Boolean
```

Comment:
This should be made mandatory i.e. `Boolean!`

Reply:
done